comp/etw/impl/etwSession.go

```diff
@@ -204,6 +240,9 @@ func createEtwSession(name string) (*etwSession, error) {
 
 	pProperties.LogFileMode = C.EVENT_TRACE_REAL_TIME_MODE
 
+	if cfg.MaxBuffers > 0 {
+		pProperties.MaximumBuffers = C.ulong(cfg.MaxBuffers)
```

Comment:
It seems like `MinimumBuffers` and `MaximumBuffers` have some interplay, should provide an option for `MinimumBuffers`, too?
https://learn.microsoft.com/en-us/windows/win32/api/evntrace/ns-evntrace-event_trace_properties

Reply:
According to linked doc
```For file-mode and real-time traces with moderate event rates, most users should minimize memory usage by setting MinimumBuffers to 0 or to a small minimum```
We're doing real-time traces.  It seemed of limited utility.  But, as you point out it could be trivially added if you think it worthwhile.

Reply:
only if you think this bit might be relevant to the kernel logger we watch, and in turn might effect what events get dropped.

> ETW cannot allocate on-demand buffers if the event is generated by a driver running at high IRQL. If your trace session needs to record events from high-IRQL kernel-mode providers, it may need to use a higher value of MinimumBuffers to force the buffers to be preallocated.

Reply:
sold